src/selectors/languages.js

```diff
@@ -1,16 +1,34 @@
 // Copyright 2019 Stanford University see LICENSE for license
+import { parseLangTag } from "utilities/Language"
 
-/**
- * Return the full name of the language given the ISO 639 language code
- * @param [Object] state
- * @param [string] languageId the identifier of the language, (e.g. 'eng')
- * @return [string] the label of the language or an empty string
- */
-export const selectLanguageLabel = (state, languageId) =>
-  state.entities.languages[languageId] || ""
+export const selectLanguageLabel = (state, tag) => {
```

Comment:
Getting the language label now requires handling the language, script, and transliteration.